Missing pytest. Lots of errors like this:
```
conftest (unittest.loader._FailedTest) ... ERROR
test (unittest.loader._FailedTest) ... ERROR
testing (unittest.loader._FailedTest) ... ERROR
joblib.test (unittest.loader._FailedTest) ... ERROR
joblib.testing (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: conftest (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: conftest
Traceback (most recent call last):
  File "/nix/store/d0im7d8gvk39xyhwnsqs57rmzvwy8g89-python3-3.6.3/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/build/joblib-0.11/doc/conftest.py", line 2, in <module>
    from joblib.test.common import np, setup_autokill, teardown_autokill
  File "/build/joblib-0.11/joblib/test/__init__.py", line 1, in <module>
    from joblib.test import test_memory
  File "/build/joblib-0.11/joblib/test/test_memory.py", line 32, in <module>
    from joblib.test.common import with_numpy, np
  File "/build/joblib-0.11/joblib/test/common.py", line 12, in <module>
    from joblib.testing import SkipTest, skipif
  File "/build/joblib-0.11/joblib/testing.py", line 12, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
```

--- a/pkgs/development/python-modules/joblib/default.nix
+++ b/pkgs/development/python-modules/joblib/default.nix
@@ -6,6 +6,7 @@
 , numpydoc
 , isPy3k
 , stdenv
+, pytest
 }:
 
 
@@ -18,14 +19,10 @@ buildPythonPackage rec {
     sha256 = "7b8fd56df36d9731a83729395ccb85a3b401f62a96255deb1a77220c00ed4085";
   };
 
-  checkInputs = [ nose sphinx numpydoc ];
+  checkInputs = [ sphinx numpydoc pytest ];
 
-  # Failing test on Python 3.x and Darwin
-  postPatch = '''' + lib.optionalString (isPy3k || stdenv.isDarwin) ''
-    sed -i -e '70,84d' joblib/test/test_format_stack.py
-    # test_nested_parallel_warnings: ValueError: Non-zero return code: -9.
-    # Not sure why but it's nix-specific. Try removing for new joblib releases.
-    rm joblib/test/test_parallel.py
+  checkPhase = ''
+    py.test -k 'not test_disk_used and not test_nested_parallel_warnings' joblib/test
   '';
 
   meta = {

